scenes/game_elements/characters/enemies/throwing_enemy/components/throwing_enemy.gd

```diff
@@ -250,6 +254,7 @@ func shoot_projectile() -> void:
 		projectile.node_to_follow = player
 	projectile.small_fx_scene = projectile_small_fx_scene
 	projectile.big_fx_scene = projectile_big_fx_scene
+	projectile.trail_fx_scene = projectile_trail_fx_scene
 	projectile.speed = projectile_speed
 	projectile.duration = projectile_duration
```

Comment:
I don't think you should change this in this PR, but, in effect what all this code is doing is papering over the fact that `PROJECTILE_SCENE` is a constant. Another approach would be to have instances of combat define an inherited scene (or a copy of `projectile.tscn`, duplicating its structure but referencing the same scripts), set properties on that, and then have the throwing enemy have a `projectile_scene: PackedScene` export.

Reply:
As an aside, I think [this](https://www.reddit.com/r/godot/comments/129sgc8/what_is_the_intended_workflow_for_inheriting/) is one of the threads that put me off using inherited scenes though:

> My advice is to avoid it if possible.

> scene inheritence is best used sparingly, but does come in handly if you have a bunch of things that are behaviorally identical but have a few different properties

>> What is the intended workflow for inheriting scenes?
>
> Not using the feature.

Reply:
You are right! I don't know how I reached this state, but the enemy should export a PackedScene for the projectile scene directly.